### ISSUE TITLE
[MU4 Issue] Accessibility: unable to reach "..." menu in Mixer by keyboard

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16066]
**Describe the bug**
The "..." menu for the Mixer is not reachable by keyboard

**To Reproduce**
1. Press F10 to open mixer
2. Try to reach the "..." menu using F6/Tab/Cursor
3. See error: it's not reachable

**Expected behavior**
Not clear what the right navigation should look like.  Probably it should be as the palettes - when you open the mixer the focus should be on the title bar then right or down cursor would take you to the "..." menu.  Or maybe there should be a different design for all subwindows.

**Platform information**
 - OS: Windows, Linux

**Additional context**
Luckily, the menu isn't a particularly important one, especially for blind users, since it is mostly about the *appearance* of the mixer.

**Comment by cbjeukendrup** [comment]
Workaround: press `P` to open the Piano keyboard; the keyboard will be tabbed together with the Mixer. The resulting tab bar _is_ keyboard accessible.

**Comment by shoogle** [comment]
> Not clear what the right navigation should look like. Probably it should be as the palettes

Yes, if you have the Timeline or Piano keyboard open as well as the Mixer then you'll see that the situation is the same as with the side panels where the palettes are, and should work the same in terms of keyboard access.

![image](https://user-images.githubusercontent.com/11011881/214707620-58c70686-becd-483c-b969-d0c3c757acc9.png)

![image](https://user-images.githubusercontent.com/11011881/214707846-9df733b2-75e5-42f5-abe7-7be8f12acebe.png)

Of course that's not to say that the side panel accessibility is perfect, but whatever the implementation is, it should be the same in both places.

**Comment by shoogle** [comment]
> Workaround: press P to open the Piano keyboard; the keyboard will be tabbed together with the Mixer. The resulting tab bar is keyboard accessible.

This doesn't appear to work for me on Windows 11. Pressing `P` doesn't change focus, and pressing Tab skips right over the bottom panel tab bar even when the Mixer and Piano keyboard are both showing.

**Comment by cbjeukendrup** [comment]
While the issue with navigation in the dock title bar in the case of a single panel is quite easy to solve, there seems to be a slightly more complicated problem with the navigation sections. Which navigation section is used, depends on the location where the panel is docked, but this dependency doesn't work. The result is that panels that are docked at the right end up having the same navigation section as panels at the left, while the navigation orders are not adjusted to have the `NavigationPanels` of two different dock panels in the same navigation section, so navigation jumps back and forth between left and right. Something similar is probably the case for horizontal panels.

**Comment by Eism** [comment]
It seems that there should be a model for each page that, depending on the location of the panels relative to each other, updates the order for the panels/sections.

**Comment by jessjwilliamson** [comment]
There will be a redesign of the bottom docking where the '...' button will be next to the name of the panel. Example when it's just Mixer open, that will be changed into its own tab instead. That's coming and should solve this problem better. Also rather annoying to go from the '...' button from the very right to then moved to the left when another tab is opened!

**Comment by DmitryArefiev** [comment]
@MarcSabatella @shoogle Please try #17439 when you have time (or nightly master tomorrow). Mixer navigation should be better than in master now. 

Also, navigation order after #17439 became a little bit different (when navigating with Tab)

**Comment by MarcSabatella** [comment]
I canb confirm I can reach the "...".  As mentioned, there are differences in the exact order of things, and I gather they are intentional.  So far I'm not noticing issues but I haven't tried every combination.  Looking forward to exploring more over time.

**Comment by DmitryArefiev** [comment]
Great! Thank you!
